### PR TITLE
Try to split base contract

### PR DESCRIPTION
**Pull request by wuminzhe**

### CODE REVIEW

**Comment by HackFisher** [comment]
@wuminzhe There are conflict

**Review by hujw77 — APPROVED** [review]
LGTM

File: contracts/periphery/contracts/s2s/baseapps/AppShare.sol [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    bytes4 internal constant DARWINIA_CHAIN_ID = 0x64617277; // darw
+    bytes4 internal constant CRAB_CHAIN_ID = 0x63726162; // crab
+    bytes4 internal constant PANGORO_CHAIN_ID = 0x70616772; // pagr
+    bytes4 internal constant PANGOLIN_CHAIN_ID = 0x7061676c; // pagl
+    bytes4 internal constant PANGOLIN_PARACHAIN_CHAIN_ID = 0x70676c70; // pglp
+    bytes4 internal constant CRAB_PARACHAIN_CHAIN_ID = 0x63726170; // crap
+
+    // Lane ids
+    bytes4 internal constant ZERO_LANE_ID = 0x00000000;
+    bytes4 internal constant ROLI_LANE_ID = 0x726f6c69; // roli
```
Comment (xiaoch05) [inline contracts/periphery/contracts/s2s/baseapps/AppShare.sol:15]: Add
pangolin<>pangoro
carb<>darwinia
crab<>crab parachain

**Review by xiaoch05 — APPROVED** [review]

File: contracts/periphery/contracts/s2s/baseapps/AppShare.sol (hunk 1 above)
Comment (wuminzhe) [reply contracts/periphery/contracts/s2s/baseapps/AppShare.sol:15]: Do the three bridges use the same 'roli' lane id?

Comment (xiaoch05) [reply contracts/periphery/contracts/s2s/baseapps/AppShare.sol:15]: No，they are different

**Pull request merged**